plaso/containers/sessions.py

```diff
@@ -19,6 +19,7 @@ class Session(interface.AttributeContainer):
     aborted (bool): True if the session was aborted.
     analysis_reports_counter (collections.Counter): number of analysis reports
         per analysis plugin.
+    artifacts_filter_file (str): path to a file containing artifact definitions.
```

Comment:
path to a file containing names of artifact definitions that are used for filtering file system and Windows Registry key paths.

Reply:
Done